test/unit/bson.test.ts

```diff
@@ -1,22 +1,21 @@
-'use strict';
+import { expect } from 'chai';
 
-const { expect } = require('chai');
-const BSON = require('../mongodb');
+import { BSON } from '../mongodb';
 
 describe('When importing BSON', function () {
   const types = [
     ['Long', 23],
     ['ObjectId', '123456789123456789123456'],
     ['Binary', Buffer.from('abc', 'ascii')],
-    ['Timestamp', 23],
+    ['Timestamp', 23n],
```

Comment:
is this an expected change in behavior? if it's a side effect of the bigint work, feels like something we should intentionally handle in the driver

Reply:
Not a side effect of bigint work, I can also change this to `{ i: number, t: number }` the Timestamp constructor got an overhaul and single number arguments are no longer permitted to avoid ambiguity